examples/linear_model/plot_poisson_regression_non_normal_loss.py

```diff
@@ -292,21 +315,40 @@ def score_estimator(estimator, df_test):
 
 ##############################################################################
 # The experimental data presents a long tail distribution for ``y``. In all
-# models we predict a mean expected value, so we will have necessarily fewer
-# extreme values. Additionally, the normal distribution used in ``Ridge`` and
-# ``RandomForestRegressor`` has a constant variance, while for the Poisson
-# distribution used in ``PoissonRegressor``, the variance is proportional to
-# the mean predicted value.
+# models, we predict the expected frequency of a random variable, so we will
+# have necessarily fewer extreme values than for the observed realizations of
+# that random variable. Additionally, the normal distribution used in ``Ridge``
+# has a constant variance, while for the Poisson distribution used in
+# ``PoissonRegressor`` and ``HistGradientBoostingRegressor``, the variance is
+# proportional to the predicted expected value.
+#
+# Thus, among the considered estimators, ``PoissonRegressor`` and
+# ````HistGradientBoostingRegressor```` are a-priori better suited for modeling
```

Comment:
```suggestion
# `HistGradientBoostingRegressor` are a-priori better suited for modeling
```

Reply:
We shall still use 2 backticks for consistency although I believe our CSS will render those in the same way.

Reply:
2 backticks is fine but this one has 4 ;)

Reply:
Yes I agree, will fix manually.